changelog.txt

```diff
@@ -0,0 +1 @@
+changed - Engines field is now required in package.json for functions deploys.
```

Comment:
@bkendall do you think this would be better as "important" instead of "changed"?

Reply:
"important - An important item (will be bolded). For example, this may be a general note about the release or a warning about a breaking change."

Reply:
Either way is fine by me. I don't think either is wrong.

Reply:
Cool - changed to "important" to highlight the breaking change.